### DISCUSSION TITLE
Cogvideo1.5 fine tuning is possible now?

### DISCUSSION THREAD

**Discussion by FCB-Sunny** [THUDM/CogVideo#554] [Q&A] [answered]
Hi, @zRzRzRzRzRzRzR Thanks for your effort and I hope to know currently it is possible to fine tune the new released model cog1.5 by using the same fine tuning script on your github repo?
And if it is possible, how can I prepare the dataset for this model? which resolution dataset is the best? and how many?
Thanks

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
Support, our main branch supports SFT and Lora CogVideoX1.5 and CogVideo1.0, for more details, please check the finetune folder. Note that this is for fine-tuning on the diffusers version.
